Add tests for register and course info screen, updates to existing tests

Divakar-Patil-Clarice opened the pull request:
Updated some of the existing tests. Added tests for register screen and course info screen.

cc @rohan-dhamal-clarice @aleffert @nasthagiri @shahidtamboli

aleffert commented on AcceptanceTests/pom.xml:
```diff
@@ -19,11 +19,11 @@
 		<deviceOS>android</deviceOS>
-		<appPath>sauce-storage:edxDebug.apk</appPath>
-		<osVersion>4.4</osVersion>
+		<appPath>/Users/divakarpatil/Desktop/edx-prod-release-1.0.04-45.apk</appPath>
```
I'm pretty sure this change is unintentional?

Divakar-Patil-Clarice replied:
Thanks. Will update this.

aleffert commented on AcceptanceTests/pom.xml:
```diff
@@ -19,11 +19,11 @@
+		<osVersion>4.4.2</osVersion>
 		<deviceName>Android Emulator</deviceName>
 		<appPackage>org.edx.mobile</appPackage>
 		<emailId>divakar.patil@claricetechnologies.com</emailId>
```
Also, I just noticed this line - which isn't part of the change, but probably your email shouldn't be hardcoded in?

Divakar-Patil-Clarice replied:
Sure, will remove all the hard coded stuff.

aleffert commented:
:+1:

Divakar-Patil-Clarice commented:
@aleffert  - I have updated the code as per comments.
I have also added new testcases for enrollment in offline mode. 

Please review - @aleffert @rohan-dhamal-clarice @nasthagiri @shahidtamboli

aleffert commented:
:+1:
